src/Calendar/Navigation.jsx

```diff
@@ -78,6 +78,14 @@ export default function Navigation({
     && maxDate <= nextActiveStartDate2
   );
 
+  // Make sure the navigation is not navigable at the first render
+  // so that the calendar takes the initial focus.
+  const [tabIndex, setTabIndex] = React.useState(-1);
+  React.useEffect(() => {
+    setTabIndex(-1);
+    setTimeout(() => setTabIndex(0), 0);
+  }, [view]);
```

Comment:
That's a breaking change - we're currently requiring React 16.3 and up. Not saying this is a bad thing, just noting this to self.

Reply:
Ah right. Well, you could make that call I guess, in the similar spirit as not supporting older browsers.